apps/ewallet_db/lib/ewallet_db/uploaders/file.ex

```diff
@@ -0,0 +1,19 @@
+defmodule EWalletDB.Uploaders.File do
+  @moduledoc """
+  Uploads an avatar after renaming it and transforming it.
+  """
+  use Arc.Definition
+  use Arc.Ecto.Definition
+
+  @acl :private
+  @versions [:original]
+
+  def validate({file, _}) do
+    ~w(.csv) |> Enum.member?(Path.extname(file.file_name))
+  end
+
+  # Override the storage directory:
+  def storage_dir(_, _) do
+    "private/uploads/#{Mix.env()}/exports"
```

Comment:
@sirn Would this work with distillery?

Reply:
Neeed to change to `Application.get_env(:ewallet, :env)`. Mix is not present in Distillery.